lib/binding_rust/lib.rs

```diff
@@ -93,6 +94,8 @@ pub enum LogType {
     Lex,
 }
 
+type TSFieldId = Option<NonZeroU16>;
```

Comment:
Let's just call this `FieldId`, instead of `TSFieldId`; I think that matches the normal rust style a bit better.

Reply:
agreed